net.christianbeier.MultiVNC.yml

```diff
@@ -0,0 +1,45 @@
+id: net.christianbeier.MultiVNC
+runtime: org.gnome.Platform # using GNOME since it contains libsecret already built
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: multivnc
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=host # for saving screenshots and logs
+modules:
+  - name: wxwidgets
+    rm-configure: true
+    config-opts:
+      - --with-gtk=3
+      - --enable-secretstore
+      - --enable-debug=no
+      - --enable-aui=no
+      - --enable-html=no
+      - --enable-mediactrl=no
+      - --enable-propgrid=no
+      - --enable-ribbon=no
+      - --enable-richtext=no
+      - --enable-stc=no
+      - --enable-xrc=no
+    cleanup:
+      - /bin
+      - /include
+      - /lib/debug
+      - /lib/wx
+      - /share/bakefile
+      - /share/aclocal
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.2.1/wxWidgets-3.2.2.1.tar.bz2
+        sha256: dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02
+  - name: multivnc
+    buildsystem: cmake
```

Comment:
```suggestion
    buildsystem: cmake-ninja
```

Reply:
Oh, that wasn't in the docs. What's the intended benefit @hfiguiere ? - faster build speeds?